src/dt_prom.c

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2023 DomainTools LLC
+ * Copyright (c) 2023-2024 DomainTools LLC
  * Copyright (c) 2008-2019, 2021 by Farsight Security, Inc.
```

Comment:
dt_prom.c and dt_prom.h should only have the 2024 DomainTools Copyright; they were newly authored this year.